src/sidebar/store/debug-middleware.js

```diff
@@ -8,14 +13,16 @@
  * to the console, along with the application state before and after the action
  * was handled.
  *
- * @param {import("redux").Store} store
+ * @param {Store} store
  */
 export function debugMiddleware(store) {
   /* eslint-disable no-console */
   let serial = 0;
 
-  return function (next) {
-    return function (action) {
+  /** @param {(a: Action) => void} next */
+  return next => {
+    /** @param {Action} action */
+    return action => {
       // @ts-ignore The window interface needs to be expanded to include this property
```

Comment:
I know this line wasn't part of the changes here, but I wondered if this was still true.

Reply:
Yes it is. `window.debug` is a custom property that we came up with.

Reply:
Ah! Yes! The comment's wording threw me. It sounded like something needed to be done externally/upstream/elsewhere ("needs to be expanded"). It didn't read as "debug is a custom property we added".